cli/cmd/cluster-apply.go

```diff
@@ -88,6 +89,24 @@ func runClusterApply(cmd *cobra.Command, args []string) {
 		ctxLogger.Fatalf("Verify cluster: %v", err)
 	}
 
+	// Update kube-system namespace with labels.
+	kubesystemNamespace := k8sutil.Namespace{
+		Name: "kube-system",
+		Labels: map[string]string{
+			internal.NamespaceLabelKey: "kube-system",
+		},
+		Annotations: map[string]string{},
+	}
+
+	cs, err := k8sutil.NewClientset(kubeconfig)
+	if err != nil {
+		ctxLogger.Fatalf("Failed to create clientset: %v", err)
+	}
+
+	nsclient := cs.CoreV1().Namespaces()
+	if err := k8sutil.CreateOrUpdateNamespace(kubesystemNamespace, nsclient); err != nil {
+		ctxLogger.Fatalf("error updating namespace %q with labels: %q", "kube-system", err)
```

Comment:
```suggestion
		ctxLogger.Fatalf("Error updating namespace %q with labels: %q", "kube-system", err)
```

Those logs are printed to the user, so they should be capitalized.